components/identity-mgt/org.wso2.carbon.identity.mgt/src/main/java/org/wso2/carbon/identity/mgt/constants/IdentityMgtConstants.java

```diff
@@ -231,6 +231,37 @@ private Notification(){
 
     }
 
+    public static class TenantManagement {
+
+        public static final String ACTION_CREATE = "CREATE";
+
+        public static final String ACTION_UPDATE = "UPDATE";
+
+        public static final String ACTION_ACTIVATE = "ACTIVATE";
+
+        public static final String ACTION_DEACTIVATE = "DEACTIVATE";
+
+        public static final String EVENT_CREATE_TENANT_URI = "https://schemas.identity.wso2.org/events/tenant/event-type/tenantCreated";
+
+        public static final String EVENT_UPDATE_TENANT_URI = "https://schemas.identity.wso2.org/events/tenant/event-type/tenantOwnerUpdated";
+
+        public static final String EVENT_ACTIVATE_TENANT_URI = "https://schemas.identity.wso2.org/events/tenant/event-type/tenantActivated";
+
+        public static final String EVENT_INITIATOR = "SYSTEM";
+
+        public static final String HEADER_PROPERTY = "header.";
+
+        public static final String USER_NAME = "username";
+
+        public static final String PASSWORD = "password";
+
+        public static final String NOTIFICATION_ENDPOINT = "notification_endpoint";
+        
+        private TenantManagement() {
+        }
+
```

Comment:
formatting

Reply:
Fixed in https://github.com/wso2/carbon-identity-framework/pull/7075/commits/3e683ea843d1e673fef01db158f76720310e92dd